sdk/loadtestservice/azure-resourcemanager-loadtestservice/src/samples/java/com/azure/resourcemanager/loadtestservice/ReadmeSamples.java

```diff
@@ -0,0 +1,193 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
```

Comment:
nit, remove this line as it is not generated.

Reply:
There is other style problem reported in this file, please check CI and fix.

Reply:
you need to have the copyright (first 2 lines), but not the 3rd about "generated"